modules/iam-role-for-service-accounts-eks/variables.tf

```diff
@@ -145,6 +145,19 @@ variable "attach_efs_csi_policy" {
   default     = false
 }
 
+# S3 CSI
+variable "attach_s3_csi_policy" {
+  description = "Determines whether to attach the EFS CSI IAM policy to the role"
```

Comment:
I think here typo EFS > S3

Reply:
Thanks for catching that. Just fixed it!